index.bs

```diff
@@ -407,4 +434,30 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
                 </ol>
             </ul>
         </ul>
+
+<h3 id='3-3-external-spec-integrations'>External Spec Integrations</h3>
+
+<h4 id='3-3-1-event-loop'>HTML Processing Model: Event Loop</h4>
+    An <a>Intersection Observer</a> processing step should take place AFTER the
+    layout and rendering steps have been performed in the
+    <a href="https://html.spec.whatwg.org/multipage/webappapis.html#processing-model-8">
+    update the rendering</a> event loop in the HTML Processing Model.
+
+    This step is:
+    <ol start="11">
+        <li><a>Run the update intersection observations steps</a> for the
+        <a href="https://html.spec.whatwg.org/multipage/webappapis.html#event-loop">
+        event loop</a> in question, passing in <i>now</i> as the timestamp.</li>
+    </ol>
+
+<h4 id='3-3-2-remove-node'>DOM4: Remove a node</h4>
```

Comment:
A monkey-patch on DOM is actually not necessary. You can just define [removing steps](https://dom.spec.whatwg.org/#concept-node-remove-ext) for all element nodes.